src/main.py

```
@@ -0,0 +1,66 @@
+            "dbo:manager": ["coached by", "under", "manager"],
+            "dbo:recipient": ["received by", "awarded to"]
+        }
+        self.embeddings = {k: self.model.encode(v) for k, v in self.predicates.items()}
+
+    def extract(self, text, subj, obj):
+        # Extract phrase between entities
+        try:
+            start = text.find(subj) + len(subj)
+            end = text.find(obj)
+            phrase = text[start:end].strip() if start < end else text
+        except: phrase = text
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)